middleware/redirects.js

```diff
@@ -60,11 +62,43 @@ function redirectNoWelsh(req, res, next) {
     }
 }
 
+/**
+ * Vanity URL lookup
+ * - First look up global/en alias from the CMS
+ * - If that fails check for a welsh specific alias
+ * - Call next() and passthrough on any failures
+ */
+async function vanityLookup(req, res, next) {
+    const findAlias = find(alias => alias.from === req.path);
+    try {
+        const enAliases = await contentApi.getAliases({ locale: 'en' });
```

Comment:
💯 for use of `await`, I'm definitely guilty of littering code with Promise callbacks.

Reply:
Eh…wouldn't go that far. This is cascade-y enough that the async/await code isn't much more concise.